src/Sarif.WorkItemFiling/WorkItemFiler.cs

```diff
@@ -35,9 +37,14 @@ public class WorkItemFiler
         /// An object that represents the system (for example, GitHub or Azure DevOps)
         /// to which the work items will be filed.
         /// </param>
-        public WorkItemFiler(FilingTarget filingTarget, IGroupingStrategy groupingStrategy, IFileSystem fileSystem)
+        public WorkItemFiler(
+            FilingTarget filingTarget,
+            IFilteringStrategy filteringStrategy,
+            IGroupingStrategy groupingStrategy,
+            IFileSystem fileSystem)
         {
             _filingTarget = filingTarget ?? throw new ArgumentNullException(nameof(filingTarget));
+            _filteringStrategy = filteringStrategy ?? throw new ArgumentNullException(nameof(filteringStrategy));
             _groupingStrategy = groupingStrategy ?? throw new ArgumentNullException(nameof(groupingStrategy));
             _fileSystem = fileSystem ?? throw new ArgumentNullException(nameof(fileSystem));
```

Comment:
>fileSystem [](start = 26, length = 10)

certainly we should be instantiating a default file system here, i.e., the one that maps to System.IO.

Reply:
Fixed.

---
In reply to: [306116240](https://github.com/microsoft/sarif-sdk/pull/1582#discussion_r306116240) [](ancestors = 306116240)